fuel-indexer/src/service.rs

```diff
@@ -123,14 +123,19 @@ async fn spawn_executor_from_index_asset_registry(
 }
 
 fn make_task<T: 'static + Executor + Send + Sync>(
-    fuel_node_addr: SocketAddr,
+    fuel_node_addr: &str,
     kill_switch: Arc<AtomicBool>,
     mut executor: T,
     start_block: Option<u64>,
 ) -> impl Future<Output = ()> {
     let mut next_cursor = None;
     let mut next_block = start_block.unwrap_or(1);
-    let client = FuelClient::from(fuel_node_addr);
+    let client = FuelClient::from_str(fuel_node_addr).unwrap_or_else(|_| {
```

Comment:
@AP-Atul Since `FuelNodeConfig` implements `ToString` maybe this PR could just be limited to:

```rust
let client = FuelClient::from_str(fuel_node_addr).unwrap_or_else(|e| panic!());
```

Just instantiating from the string instead of the socket?